doc/user/compatibility.md

```diff
@@ -212,14 +212,8 @@ provide the same interface as JRuby does for this functionality.
 
 #### Java to Ruby interop
 
-Calling Ruby code from Java is supported by the `PolyglotEngine` API of Truffle.
-
-http://lafo.ssw.uni-linz.ac.at/javadoc/truffle/latest/com/oracle/truffle/api/vm/PolyglotEngine.html
-
-In the future we hope to also support the standard JSR 223, Scripting for the
-Java Platform, specification and the `ScriptEngineManager` class, which is also
-supported by JRuby. We are unlikely to ever implement JRuby's own scripting
-interface such as the `Ruby` and `RubyInstanceConfig` classes.
+Calling Ruby code from Java is supported by the
+[GraalVM polyglot API](http://www.graalvm.org/truffle/javadoc/).
```

Comment:
Should the link be more specific and point to documentation showing this?

Reply:
Isn't this called the `Context` API? (GraalVM seems unrelated here at least)

Reply:
It's being branded as part of GraalVM - the package is literally `org.graalvm`. I didn't want to get into too many specifics, I didn't want to duplicate and maintain documentation for the SDK. That's something they can do centrally - we just have to point at it.

Reply:
I would just say the `Polyglot API`.
`org.graalvm` is just the base package name of graal/truffle/sdk/etc and does not seem to have any real relation to GraalVM the release artifact (except done by the same people).

Reply:
(Truffle actually still has the old package named for API compatibility)

Reply:
It's not a package name change, it's a totally different API and the old one is going to be deprecated.

Reply:
Apparently the proper name is 'Graal-SDK polyglot API'.